stable/kube-bench/values.yaml

```diff
@@ -5,7 +5,7 @@ cronjob:
 
 image:
   repository: aquasec/kube-bench
-  tag: 0.1.0
+  tag: latest
```

Comment:
`latest` can lead to many problems. Can we use a hash if there's no proper tag available?